03-single-cell-with-bioconductor.Rmd

```diff
@@ -139,17 +134,145 @@ library(iSEE)
 
 First we will take a look at the interactive plots that iSEE can display.
 
```

Comment:
Do we need to say anything about the interactive plots? This section feels a little barebones, but I know we've limited time for the actual presentation.

Reply:
I was thinking we'd just scroll around on the dashboard. Do we want to do more than that?

Reply:
No, that makes sense to me.